std_msgs/Header header

geometry_msgs/Polygon safety1
geometry_msgs/Polygon safety2
geometry_msgs/Polygon safety3
geometry_msgs/Polygon warn1
geometry_msgs/Polygon warn2
geometry_msgs/Polygon muting1
geometry_msgs/Polygon muting2

geometry_msgs/Polygon safety1_sub0
geometry_msgs/Polygon safety2_sub0
geometry_msgs/Polygon safety3_sub0
geometry_msgs/Polygon warn1_sub0
geometry_msgs/Polygon warn2_sub0
geometry_msgs/Polygon muting1_sub0
geometry_msgs/Polygon muting2_sub0

geometry_msgs/Polygon safety1_sub1
geometry_msgs/Polygon safety2_sub1
geometry_msgs/Polygon safety3_sub1
geometry_msgs/Polygon warn1_sub1
geometry_msgs/Polygon warn2_sub1
geometry_msgs/Polygon muting1_sub1
geometry_msgs/Polygon muting2_sub1

geometry_msgs/Polygon safety1_sub2
geometry_msgs/Polygon safety2_sub2
geometry_msgs/Polygon safety3_sub2
geometry_msgs/Polygon warn1_sub2
geometry_msgs/Polygon warn2_sub2
geometry_msgs/Polygon muting1_sub2
geometry_msgs/Polygon muting2_sub2

float32 speed_lower
float32 speed_upper
